packages/core/src/Coordinator.js

```diff
@@ -134,18 +134,20 @@ export class Coordinator {
    *  or a SQL string.
    * @param {object} [options] An options object.
    * @param {'arrow' | 'json'} [options.type] The query result format type.
-   * @param {boolean} [options.cache=true] If true, cache the query result.
+   * @param {boolean} [options.cache=true] If true, cache the query result in the client (QueryManager).
+   * @param {boolean} [options.persist=false] If true, persist cached query result in the server.
    * @param {number} [options.priority] The query priority, defaults to
    *  `Priority.Normal`.
    * @returns {QueryResult} A query result promise.
    */
   query(query, {
     type = 'arrow',
     cache = true,
+    persist = false,
```

Comment:
The default value of `persist` differs depending on the server implementation
- [`duckdb-server-rust`: `true`](https://github.com/uwdata/mosaic/blob/57580776bcf69a37b6c3c0724a970d4db7d8a7fd/packages/duckdb-server-rust/src/query.rs#L18) cc @domoritz 
- [`duckdb-server` (Python): `false`](https://github.com/uwdata/mosaic/blob/57580776bcf69a37b6c3c0724a970d4db7d8a7fd/packages/duckdb-server/pkg/query.py#L24)
- [`duckdb` (Node.js): `false`](https://github.com/uwdata/mosaic/blob/57580776bcf69a37b6c3c0724a970d4db7d8a7fd/packages/duckdb/src/Cache.js#L52)

Reply:
`persist = false,` has been removed as it needs to be part of `...options`.
But that means the default value of `persist` should be the same across the servers